src/Lykke.Quintessence.DomainServices/DefaultBlockchainService.cs

```diff
@@ -202,7 +202,7 @@ public virtual async Task<BigInteger> GetBestTrustedBlockNumberAsync()
                     transferAmount: amount
                 );
             }
-            catch (GasAmountEstimationException)
+            catch (Exception)
```

Comment:
It is bad to catch everything, you know that. If the node is not available fail fast, am I wrong?